### ISSUE TITLE
Add option to select output device

### ISSUE THREAD

**Issue by ghost** [lukedirtwalker/cutespotify#8]
Hi, could you add a button which would allow to change output to bluetooth speakers or headphones. And in cases when bt is not activated, the cutespotify would activate it and connect to selected source.
Normally I have bluetooth disabled and I only activate it when I want to connect to bt speakers.

Thanks!

**Comment by lukedirtwalker** [comment]
The system itself should manage this already, if you pair with the bluetooth speaker.

Anyway I don't have a bluetooth device to test, so at the moment I can't help with this.

But feel free to add it yourself.

**Comment by ghost** [comment]
yeah, the system manages it, but it would be faster to connect to bt device by having button in cutespotify:)
